frontend/src/config/i18n/en/seeds.json

```diff
@@ -35,5 +41,6 @@
   "yield": "Yield",
   "price": "Price",
   "generation": "Generation",
-  "notes": "Notes"
+  "notes": "Notes",
+  "error_modal_title": "Fehler"

```

Comment:
Shouldn't be needed if we use a toast?

Reply:
I use this as a placeholder in case there was an error fetching a plants name.